app/src/page/file/view.js

```diff
@@ -220,6 +222,31 @@ class FilePage extends React.PureComponent {
     }
   }
 
+  onMediaLoaded = (title, channelName) => {
+    this.setState({ mediaLoaded: true });
+    window.currentMediaInfo = {
+      title: title,
+      channel: channelName
+    };
+  }
+
+  onPlaybackStarted = () => {
+    let timeToStart = null;
+    if (this.startTime) {
+      timeToStart = Math.ceil((Date.now() - this.startTime) / 1000);
+      this.startTime = null;
+    }
+
+    const { navigation } = this.props;
+    const { uri } = navigation.state.params;
+    let payload = { 'Uri': uri };
+    if (!isNaN(timeToStart)) {
+      payload['Time to Start (seconds)'] = timeToStart;
+    }
+
+    NativeModules.Mixpanel.track('Play', payload);
```

Comment:
`time_to_start` should be using internal-apis.
https://github.com/lbryio/lbry-desktop/blob/master/src/renderer/component/fileViewer/view.jsx#L186

It also needs to be `ms`.